Makefile

```diff
@@ -38,11 +42,20 @@ endif
 ifeq ($(RELEASE_GPG_KEYNAME),)
 	$(error RELEASE_GPG_KEYNAME must be set to build a release and deploy this package)
 endif
-	@echo "==> Tagging version $(VERSION)"
-	@bash ./build/publish $(VERSION) validate
-	@git tag --sign -a "$(VERSION)" -m "riak_pb $(VERSION)" --local-user "$(RELEASE_GPG_KEYNAME)"
-	@git push --tags
-	@bash ./build/publish $(VERSION)
+ifeq ($(MIX_VERSION),)
+	$(error The mix command is required to publish to hex.pm)
+endif
+	echo "==> Tagging version $(VERSION)"
+	$(PROJDIR)/build/publish $(VERSION) validate
+	echo -n "$(VERSION)" > VERSION
+	git add --force VERSION
+	git commit --message="riak_pb $(VERSION)"
+	git push
+	git tag --sign -a "$(VERSION)" -m "riak_pb $(VERSION)" --local-user "$(RELEASE_GPG_KEYNAME)"
+	git push --tags
+	$(PROJDIR)/build/publish $(VERSION)
```

Comment:
Looks like at least there are some comments in the `publish` script which mention the requirement of an `X.Y.Z.N` version string